### DISCUSSION TITLE
Repository problem with my one page website........

### DISCUSSION THREAD

**Discussion by lildesire** [community/community#28613] [New to GitHub]
hello im a 0 in coding and i dont understand anything, tried for several hours to connect this one page simple website to a repository page online! if someone can do it for me i would really appreciate it! i followedall the steps on youtube but my project does not seem to work in fact i cannot even see the hello world page... not able to upload the files on this thred, i will send them in your email if you want!

**Comment by radar-blip** [comment]
Which one are you working on?

Is this what you're looking for?  https://lildesire.github.io/website001/

**Reply by lildesire** [reply]
im working on the 004 i uploaded the fles but it does not work, all files are there tho

**Reply by radar-blip** [reply]
It looks like its working now. Keep going!

**Reply by radar-blip** [reply]
I'm kinda confused by all those files. Where is this 'Hello Word' file at? What is it called?

For future reference, all your files should be lowercase. (home.html not Home.html) You will learn this. 😵‍💫

**Reply by lildesire** [reply]
its still not working and tried to upload only the "page1" on the website005 one and its not showing up

**Reply by airtower-luna** [reply]
It loads just fine, aside from not being able to find the stylesheet and various images you'd still need to upload: https://lildesire.github.io/website005/Page-1.html

If you want that page to load when you go to the site URL, you need to name the file `index.html`. What happens right now is that because there is no `index.html` the default pages build turns the README into the index file.

Also I highly recommend you stop creating a new repository for each version, and instead update one. That's what Git is for. :wink:

**Reply by jfullstackdev** [reply]
I think we now see it, 

![image](https://user-images.githubusercontent.com/47092464/183558274-a5a505d2-d7cc-460a-ab58-fa1fb556d46a.png)

see all the errors ? that's why all designs and the looks were not applied, it's about getting the correct URLs of those

**Reply by radar-blip** [reply]
@lildesire You only need 1 repository for your entire website. It makes it hard for us to help if you keep creating more repos. We dont know which one to help with. 
Its good that you're experimenting, but try and stick with one repo.

**Comment by jfullstackdev** [comment]
seems it's working by now 

![image](https://user-images.githubusercontent.com/47092464/183275322-f3663531-333d-44e6-a18c-7e1177228234.png)

**Reply by lildesire** [reply]
i will link pictures of what i shoukd look like
![Screenshot (75)](https://user-images.githubusercontent.com/85130792/183297820-90444ca9-f1fc-45bb-a545-80ee75f9e0f2.png)
![Screenshot (76)](https://user-images.githubusercontent.com/85130792/183297821-50d23466-c91d-40ea-8c51-e4b65cb7dde7.png)

**Comment by lildesire** [comment]
i will try to have the website working since it shows up but its all broken theres the big picture missing, then i will re upload if it doesnt work i will reply here, also thank you for helping me! 👍🏻

On Aug 7, 2022, at 9:07 AM, Shiva Shirsath ***@***.***> wrote:

﻿

Upload your website file by file to your repository

Must include index.html file

Then go to your Repository
Then go to settings of your repository
Then go to pages
select github pages
Then select branch (main/master)
Then select root
Then click save & wait 1 minit & refresh
Then you will see a link
Then click it
& boom
Your website is uploaded.

if having truble
Reply this quote

**Comment by lildesire** [comment]
hello i found my page thats working but its not the index page....   
i placed them in the drive! 
https://drive.google.com/drive/folders/16TSaFrrC2yG1nBlCWGY2t8GXO8LwxbVq?usp=sharing

**Reply by airtower-luna** [reply]
Just link the repository and site URL, that's usually the best way way to get help. Missing pictures are usually due to broken links, but it's hard to be sure without seeing the exact code.

**Comment by jfullstackdev** [comment]
as I was checking it, I think you are not yet committing it, it's just there at your computer
